src/pages/workspaces/List.js

```diff
@@ -24,16 +24,15 @@ import ShareWorkspaceModal from 'src/pages/workspaces/workspace/ShareWorkspaceMo
 const styles = {
   cardContainer: listView => ({
     position: 'relative',
-    display: 'flex', flexWrap: listView ? undefined : 'wrap', marginRight: listView ? undefined : '-1rem'
+    display: 'flex', flexWrap: listView ? undefined : 'wrap',
+    marginRight: listView ? undefined : '-1rem'
```

Comment:
These negative margins allow the cards to go right up to the edge of the container, instead of line breaking early.